internal/cli/atlas/serverless/list.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2021 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package serverless
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/cli/require"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+var listTemplate = `ID	NAME	MDB VER	STATE{{range .Results}}
+{{.ID}}	{{.Name}}	{{.MongoDBVersion}}	{{.StateName}}{{end}}
+`
+
+type ListOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	cli.ListOpts
+	store store.ServerlessLister
+}
+
+func (opts *ListOpts) initStore() error {
+	var err error
+	opts.store, err = store.New(store.AuthenticatedPreset(config.Default()))
+	return err
```

Comment:
[nit - non-blocking] afaik, in mcli prefer assigning variables directly (skip declaration part)
```suggestion
	opts.store, err := store.New(store.AuthenticatedPreset(config.Default()))
	return err
```

Reply:
I've copied this around if change to `:=` I can't use `opts.store` I would have to change to something like this:
```suggestion
	store, err := store.New(store.AuthenticatedPreset(config.Default()))
	opts.store = store
	return err
```

Reply:
oh, then leave it as it is. Thank you for looking into it.